API/Repositories/Implementations/TournamentsRepository.cs

```diff
@@ -23,7 +23,14 @@ public async Task<IEnumerable<Tournament>> SearchAsync(string name)
     {
         //_ is a wildcard character in psql so it needs to have an escape character added in front of it.
         name = name.Replace("_", @"\_");
-        return await _context.Tournaments.Where(x => EF.Functions.ILike(x.Name, $"%{name}%", @"\")).ToListAsync();
+
+        List<Tournament> nameSearchResult =
+        await _context.Tournaments.Where(x => EF.Functions.ILike(x.Name, $"%{name}%", @"\")).ToListAsync();
+
+        List<Tournament> abbreviationSearchResult =
+        await _context.Tournaments.Where(x => EF.Functions.ILike(x.Abbreviation, $"%{name}%", @"\")).ToListAsync();
+
+        return nameSearchResult.Union(abbreviationSearchResult);
```

Comment:
This is making two round trips when we don't need to. Let's combine these into something like this:
```cs
return await _context.Tournaments.Where(x => *ILike for name* || *ILike for abbreviation*);```